src/server.ts

```diff
@@ -43,7 +45,30 @@ const clientConnectionHandler = (socket: Socket): void => {
   })
 }
 
+const shutDown = (eventHistory: Array<HistoryEvent>): void => {
```

Comment:
Could this be async so we could write:
```TypeScript
try {
  await fs.writeFile(...)
} catch(err) {
  Log.error(...)
}
```
🤓

Reply:
changed 👍